bcda/suppression/suppression_test.go

```diff
@@ -200,26 +238,35 @@ func (s *SuppressionTestSuite) TestGetSuppressionFileMetadata() {
 	testUtils.ResetFiles(s.Suite, filePath)
 	err = filepath.Walk(filePath, getSuppressionFileMetadata(&suppresslist, &skipped))
 	assert.Nil(err)
-	modtimeBefore := suppresslist[0].deliveryDate
 	modtimeAfter := time.Now().Truncate(time.Second)
-	for _, suppress := range suppresslist {
-		assert.Equal(modtimeBefore.Format("010203040506"), suppress.deliveryDate.Format("010203040506"))
-
-		// change the modification time for all the files
-		err := os.Chtimes(suppress.filePath, modtimeAfter, modtimeAfter)
-		if err != nil {
-			s.FailNow("Failed to change modified time for file", err)
-		}
+	assert.Len(suppresslist, 2)
```

Comment:
Oh was this broken out to know "which" file failed the test for the modification time change? Like better debugging if a failure happens??

Reply:
Yes. With the additional test suppression file, I updated this to check both. Also changed this assertion that was comparing a value to itself:
```
modtimeBefore := suppresslist[0].deliveryDate
...
assert.Equal(modtimeBefore.Format("010203040506"), suppress.deliveryDate.Format("010203040506"))
```

Reply:
oh crap! good find